language/move-vm/runtime/src/native_functions.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
native functions exposed live inside the Move VM and the are private to the VM.
The function themselves can come from different crates, but the MoveVM controls what it wants to expose